### PR TITLE
QA-134 Add Isomorphic Deployment ability to core smoke tests

### PR DESCRIPTION
**Pull request by tateexon**
These changes add the dockerfiles and scripts needed to use the new remote running strategy. This greatly reduces the number of port forwards our tests use, which has been a pain point for test intermittency. This does not affect how we run non smoke tests although we do plan on updating the rest to use the new pattern in CI.

### CODE REVIEW

File: integration-tests/test.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ARG BASE_IMAGE
+ARG IMAGE_VERSION=latest
+FROM ${BASE_IMAGE}:${IMAGE_VERSION}
+
+ARG SUITES=chaos migration performance reorg smoke soak
```
Comment (anirudhwarrier) [inline integration-tests/test.Dockerfile:5]: what about the benchmark suite?

**Review by kalverra — CHANGES_REQUESTED** [review]
Nits

File: integration-tests/actions/actions.go [hunk 2]
```diff
@@ -230,21 +229,16 @@ func TeardownSuite(
...
-		keepEnvs = "NEVER"
-	}
-
 	if err := testreporters.WriteTeardownLogs(t, env, optionalTestReporter); err != nil {
 		return errors.Wrap(err, "Error dumping environment logs, leaving environment running for manual retrieval")
 	}
 	for _, c := range clients {
 		if c != nil && chainlinkNodes != nil && len(chainlinkNodes) > 0 {
 			if err := returnFunds(chainlinkNodes, c); err != nil {
+				fmt.Println(environment.FAILED_FUND_RETURN)
```
Comment (AnieeG) [inline integration-tests/actions/actions.go:238]: Is it necessary?

File: integration-tests/actions/actions.go [hunk 3]
```diff
@@ -258,20 +252,7 @@ func TeardownSuite(
...
-			return env.Shutdown()
-		}
-	case "NEVER":
-		return env.Shutdown()
-	default:
-		log.Warn().Str("Invalid Keep Value", keepEnvs).
-			Msg("Invalid 'keep_environments' value, see the KEEP_ENVIRONMENTS env var")
-	}
-	return nil
+	return env.Shutdown()
```
Comment (AnieeG) [inline integration-tests/actions/actions.go:255]: do we not need `keepEnvs` logic anymore?

File: integration-tests/actions/actions.go (hunk 2 above)
Comment (tateexon) [reply integration-tests/actions/actions.go:238]: Yes this is necessary and propagates a fund return failure out of the remote runner to stop the cleanup from cleaning up the environment.

File: integration-tests/actions/actions.go (hunk 3 above)
Comment (tateexon) [reply integration-tests/actions/actions.go:255]: The keepEnvs logic now exists in chainlink-env https://github.com/smartcontractkit/chainlink-env/pull/215/files#diff-6c86ee279425279a568c6320829a514721c13cb2227ff0014f5fc75fd28ce5b1R543-R580

File: integration-tests/actions/actions.go (hunk 2 above)
Comment (tateexon) [reply integration-tests/actions/actions.go:240]: Added a comment to explain the reason for it

**Review by kalverra — DISMISSED** [review]

**Review by AnieeG — APPROVED** [review]

**Pull request merged**